tests/unit/transformers/test_categorical.py

```diff
@@ -2375,24 +2375,14 @@ def test__fit_error(self):
         """Test the ``_fit`` method checks that data is in ``self.order``.
 
         If the data being fit is not in ``self.order`` an error should be raised.
-
-        Setup:
-            - create an instance of the ``OrderedLabelEncoder``.
-
-        Input:
-            - a pandas series.
-
-        Side effects:
-            - set the ``values_to_categories`` dictionary to the appropriate value.
-            - set ``categories_to_values`` dictionary to the appropriate value.
         """
         # Setup
-        data = pd.Series([1, 2, 3, 2, 1, 4])
```

Comment:
why did we change these from ints? Shouldn't the OrderedLabelEncoder also be able to work with ints as categories (eg. enums)?

Reply:
Yes, it's still working; I can keep them `ints`. I had to make a change here because the error/warning messages were different, with numpy 2.0 the categories were no longer `3` and `4` but `np.int32(3)`, `np.int32(4)`

Reply:
I think we can leave this as long as there are tests somewhere that test ints. If not, we should make at least some of these ints

Reply:
Yes there are these tests:
https://github.com/sdv-dev/RDT/blob/87fe0103565ab11921006399443d6be57839e3b2/tests/unit/transformers/test_categorical.py#L499
https://github.com/sdv-dev/RDT/blob/87fe0103565ab11921006399443d6be57839e3b2/tests/unit/transformers/test_categorical.py#L2340